octobot_tentacles_manager/creators/tentacle_creator.py

```diff
@@ -112,22 +114,22 @@ def __init__(self, config, tentacle_type, tentacle_creator):
         self.tests = []
 
         self.config_file = self.get_config_path() if self.subtype in self.tentacle_creator.get_config_templates() else[]
-        self.logger = logging.getLogger(self.__class__.__name__)
-        self.logger.setLevel(INFO)
+        self.logger = get_logger(self.__class__.__name__)
+        set_global_logger_level(INFO)
 
     def get_path(self):
-        return f"{TENTACLES_PATH}/{self.t_type}/{self.subtype}/{EVALUATOR_ADVANCED_FOLDER}/{self.name}.py"
+        return f"{TENTACLES_PATH}/{self.t_type}/{self.subtype}/{self.name}.py"
 
     def get_config_path(self):
-        return f"{TENTACLES_PATH}/{self.t_type}/{self.subtype}/{EVALUATOR_CONFIG_FOLDER}/{self.name}{CONFIG_FILE_EXT}"
+        return f"{TENTACLES_PATH}/{self.t_type}/{self.subtype}/{TENTACLE_CONFIG}/{self.name}{CONFIG_EXT}"
```

Comment:
`os.path.join`

Reply:
About creator, I did not change the code, I think we need a complete refactor of it.